client/lcd/root.go

```diff
@@ -71,7 +71,8 @@ func (rs *RestServer) Start(listenAddr string, maxOpen int, readTimeout, writeTi
 
 	var h http.Handler = rs.Mux
 	if cors {
-		h = handlers.CORS()(h)
+		allowAllCORS := handlers.CORS(handlers.AllowedHeaders([]string{"Content-Type"}))
```

Comment:
This overrised all default entries (i.e. we lose them)

Reply:
The docs (which I just linked above) say: ` This is an append operation so the headers Accept, Accept-Language, and Content-Language are always allowed`

Reply:
Oh wow, what a terrible API. Thanks